sdk/servicebus/azure-servicebus/azure/servicebus/management/_models.py

```diff
@@ -196,46 +247,81 @@ class QueueRuntimeInfo(object):
     :type updated_at: ~datetime.datetime
     :ivar size_in_bytes: The size of the queue, in bytes.
     :type size_in_bytes: int
-    :ivar message_count: The number of messages in the queue.
-    :type message_count: int
-    :ivar message_count_details: Details about the message counts in entity.
-    :type message_count_details: ~azure.servicebus.management.MessageCountDetails
+    :ivar total_message_count: Total number of messages.
+    :type total_message_count: int
+    :ivar active_message_count: Number of active messages in the queue, topic, or subscription.
+    :type active_message_count: int
+    :ivar dead_letter_message_count: Number of messages that are dead lettered.
+    :type dead_letter_message_count: int
+    :ivar scheduled_message_count: Number of scheduled messages.
+    :type scheduled_message_count: int
+    :ivar transfer_dead_letter_message_count: Number of messages transferred into dead letters.
+    :type transfer_dead_letter_message_count: int
+    :ivar transfer_message_count: Number of messages transferred to another queue, topic, or
+     subscription.
+    :type transfer_message_count: int
```

Comment:
if I'm understanding it correctly, those are properties on the QueueRuntimeProperties object, not instance variables any more, so we don't need to mention them in the doc string.